src/Common/src/Interop/Linux/libc/Interop.inotify.cs

```diff
@@ -32,7 +32,7 @@ internal static int inotify_rm_watch(int fd, int wd)
                 }
                 else
                 {
-                    System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
+                    global::System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
```

Comment:
It feels a bit excessively long, personally.  I'd use something like `Interop.Libraries.CryptoShim` for `Interop.Crypto`.

But the total line length has a practical limit, and they should all be the same within a single interop file, so that's more an "I wouldn't have done it that way" than "I object".

Mainly I feel like SystemSecurityCryptographyNative is too long of an identifier, my brain's tokenizer doesn't quite have that much space, so it bogs down in parsing (or lexing, whatever).